lib/committee.ex

```diff
@@ -1,9 +1,16 @@
 defmodule Committee do
   @hooks ~w(pre_commit post_commit)
+  @version "1"
 
-  def __hooks__ do
-    @hooks
-  end
+  @doc """
+  Return a list of the currently supported hooks.
+  """
+  def __hooks__, do: @hooks
+
+  @doc """
+  Return the current version of Committee
```

Comment:
```suggestion
  Return the current version of Committee hook file format
```
WDYT? It will be different than the actual version of the package, right?

Reply:
Ah I see, yeah you're right it will be different. I'll update the copy